packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageCanvas.js

```diff
@@ -0,0 +1,141 @@
+import { arrayOf, bool, func, number, shape, string } from 'prop-types'
+import { useRef } from 'react'
+
+import SVGContext from '@plugins/drawingTools/shared/SVGContext'
+
+import InteractionLayer from '../InteractionLayer'
+
+import SVGImage from '../SVGComponents/SVGImage'
+
+function calculateAdjustedViewBox({ naturalWidth, naturalHeight, transformMatrix }) {
```

Comment:
I think this calculation may need additional work, specifically to handle zoomed out panning. When zooming out, then panning the image jumps around erratically. I'm looking into the issue, but suggestions welcome.